lib/components/T3MediaFile/type/Youtube.vue

```diff
@@ -1,12 +1,11 @@
-<template functional>
+<template>
   <iframe
-    v-if="props.file.publicUrl"
+    v-if="file.publicUrl"
     id="ytplayer"
-    v-bind="data.attrs"
-    :class="[data.class, data.staticClass]"
-    :width="props.file.properties.dimensions.width || 640"
-    :height="props.file.properties.dimensions.height || 360"
-    :src="props.file.publicUrl"
+    v-bind="$attrs"
```

Comment:
is there a need to use v-bind="$attrs" since we don't use inheritAttrs:false here ?

Reply:
Yes, definitely you are right. The thing is I had a local environment bug which shows me that the attributes are not propagating to children, so I have added the v-bind just to be sure everything work fine, but now after cleaning up local env, it's sure that we don't need that v-bind at all.